circle.yml

```diff
@@ -9,12 +9,11 @@ dependencies:
   override:
     # Used by container_spec
     - docker pull alpine
-    - docker build -t=$PRIVATE_REGISTRY/$CIRCLE_PROJECT_REPONAME:b$CIRCLE_BUILD_NUM .
-    - bundle install
+    - make image
 
 test:
   override:
-    - bundle exec rake
+    - make test
```

Comment:
If we're going that direction that much, I would propose calling the task `test_only`, and then declaring `test` as just `test: image test_only` with no other content.



~will out.

On Dec 11, 2015, 10:15 -0500, Devon Blandin<notifications@github.com>, wrote:
>  
> Incircle.yml(https://github.com/codeclimate/codeclimate/pull/265#discussion_r47364882):
>  
> >>test:>override:>- - bundle exec rake>+ - make test
>  
> All good points. What do you think of a separate Makefile command calledcitestthat does not have theimagedependency?
>  
>  
> —
> Reply to this email directly orview it on GitHub(https://github.com/codeclimate/codeclimate/pull/265/files#r47364882).
>  
>  
>